03_simon_says/simon_says.rb

```diff
@@ -1 +1,63 @@
 #write your code here
+def echo(string)
+    return string
+end
+
+def shout(string)
+    return string.upcase
+end
+
+
+# def repeat(string)
+#     return "#{string} #{string}"
+# end
+
+def repeat(string, num = 2)
+    repeating = Array.new(num, string).join(" ")
+    return repeating
+end
+
+def start_of_word(string, num)
+    return string[0..(num-1)]
+end
+
+def first_word(string)
+    i = 0
+    word = ""
+    while string[i] != " " && i < string.length
+        word += string[i]
+        i += 1
+    end
+    return word
+end
+
+
+#split into an array on the spaces
+#capitalize each index in array
+#combine array indices into string
+#return string
+
+
+#There's got to be a better way to do this, but it works
```

Comment:
So more efficient way is to  initialize a variable like `little_words=["a","and","the","over"] ` and then use the keyword `little_words.include?` to filter out certain words